cli/src/commands/init.rs

```diff
@@ -0,0 +1,51 @@
+//! Subcommand `phylum init`.
+
+use std::io;
+
+use clap::ArgMatches;
+use dialoguer::theme::ColorfulTheme;
+use dialoguer::{Confirm, Input};
+
+use crate::api::PhylumApi;
+use crate::commands::{project, CommandResult};
+
+/// Handle `phylum init` subcommand.
+pub async fn handle_init(api: &mut PhylumApi, matches: &ArgMatches) -> CommandResult {
+    let cli_project = matches.get_one::<String>("project");
+    let cli_group = matches.get_one::<String>("group");
+
+    let (project, group) = match cli_project {
+        Some(project) => (project.clone(), cli_group.cloned()),
```

Comment:
Why is the `clone()` on `project` necessary? I see only a reference to it is used afterwards and it doesn't look like it's getting moved elsewhere, but I may be missing something or it may be related to `project`'s type.

Reply:
Just because I can't get a `&String` back from prompt_project, so branch consistency basically.

I could wrap it in a cow if we really wanted to, but that's just pointless complexity really for a command that executes interactively in a couple of seconds once.

Reply:
I honestly wouldn't mind `clone`s even in hot paths as optimizations _should_ take care of that anyway more often than not, so that's fine.

Reply:
Getting `clone`s optimized away is extremely rare, so I wouldn't rely on that *at all*. But yeah in the CLI we don't really have any hot paths anyway.